compiler-rt/lib/ubsan/ubsan_handlers.cpp

```diff
@@ -141,10 +141,24 @@ void __ubsan::__ubsan_handle_type_mismatch_v1(TypeMismatchData *Data,
   GET_REPORT_OPTIONS(false);
   handleTypeMismatchImpl(Data, Pointer, Opts);
 }
+
```

Comment:
Please add a single `.` alone on a line after the `#include`. This seems to be the new [coding convention](https://git.tukaani.org/?p=xz.git;a=commitdiff;h=f9cf4c05edd14dedfe63833f8ccbe41b55823b00).

Reply:
too soon.